<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plugin Tester</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Plugin</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Plugin:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditPlugin"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QTableView" name="tableView">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Action:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditAction"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>&amp;Process</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Watch Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonAddWatch">
          <property name="text">
           <string>Add Property</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditWatch"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tableViewProp"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Property Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditPropName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Value:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditPropValue"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonGet">
        <property name="text">
         <string>Get</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonSet">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
